src/test/javascript/spec/service/faq.service.spec.ts

```
@@ -134,6 +134,26 @@ describe('Faq Service', () => {
+        it('should find faqs by courseId and status', () => {
+            const category = {
+                color: '#6ae8ac',
+                category: 'category1',
+            } as FaqCategory;
+            const returnedFromService = [{ ...elemDefault, categories: [JSON.stringify(category)] }];
+            const expected = [{ ...elemDefault, categories: [new FaqCategory('category1', '#6ae8ac')] }];
+            const courseId = 1;
+            service
+                .findAllByCourseIdAndState(courseId, FaqState.ACCEPTED)
+                .pipe(take(1))
+                .subscribe((resp) => (expectedResult = resp));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)